### DISCUSSION TITLE
Cannot get fitted_values of cross_validation if max_horizon is not None

### DISCUSSION THREAD

**Discussion by elisevansartefact** [Nixtla/mlforecast#193] [Q&A]
Suppose I have a forecast horizon N. I have noticed that if, instead of the default recursive prediction method, I specify max_horizon = N to train N models - i.e. one per horizon - then I am unable to use the fitted_values = True parameter. It leads to the error:

```
    670 trainX = trainX[self.ts.features_order_]
    671 for name, model in self.models_.items():
--> 672     insample_results[name] = model.predict(trainX)  # type: ignore[union-attr]
    673 if self.ts.target_transforms is not None:
    674     for tfm in self.ts.target_transforms[::-1]:

AttributeError: 'list' object has no attribute 'predict'
```

If I do not specify `max_horizon` then this does not happen - it works as expected and I can recover the fitted values model.cross_validation_fitted_values() with no problem.

Is there another way to get the fitted cross_validation values if I train >1 model?

**Comment by jmoralez** [comment]
Hey, thanks for raising this. This was an oversight on my side, I'll push a fix soon.
